README.md

```diff
@@ -109,6 +109,16 @@ module.exports = function (fastify, options, next) {
 $ fastify start plugin.js -- --one
 ```
 
+Modules in EcmaScript Module format can be used on Node.js >= 14.0.0
+```js
+// plugin.mjs
+const plugin = function (fastify, options, next) {
+  fastify.get('/', (req, reply) => reply.send(options))
+  next()
+}
+export { plugin as default }
```

Comment:
I prefer `export default plugin`

Reply:
Fine, this was what VScode created on conversion to Module format.

Reply:
I have pushed this in c3e1ab7

Reply:
and fixed 'examples/ts-plugin-with-custom-options.mjs' in 324d47a